src/loader/loader.js

```diff
@@ -274,7 +278,13 @@ export default class Loader {
 
 						this._setModuleImplementations(undefinedModuleNames);
 
-						success(...this._getModuleImplementations(moduleNames));
+						try {
+							success(
+								...this._getModuleImplementations(moduleNames)
```

Comment:
I don't think there is anything in `_getModuleImplementations()` that could throw right now, but it still might be worth future-proofing this by moving the call outside the `try`.

Reply:
Added feedback to the PR.